integration-cli/docker_cli_run_unix_test.go

```diff
@@ -315,3 +318,24 @@ func (s *DockerSuite) TestRunWithSwappinessInvalid(c *check.C) {
 		c.Fatalf("failed. test was able to set invalid value, output: %q", out)
 	}
 }
+
+func (s *DockerSuite) TestRunInvalidCpusetCpusFlagValue(c *check.C) {
+	testRequires(c, cgroupCpuset)
+
+	sysInfo := sysinfo.New(true)
```

Comment:
I'm using `sysinfo` just to get available cpus on the system running the test. This is needed for this integration test